lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM/SharedToDotOperandMMAv2.cpp

```diff
@@ -98,8 +99,8 @@ class MMA16816SmemLoader {
 };
 
 SmallVector<Value>
-MMA16816SmemLoader::computeLdmatrixMatOffs(Value warpId, Value lane,
-                                           Value cSwizzleOffset) {
+MMA16816SmemLoader::computeLdmatrixMatOffs(Value warpB, Value warpId,
```

Comment:
Maybe there's a cleaner way than passing warpB and warpId (which represents warpM/warpN)